src/test/regress/expected/aggregate_support.out

```diff
@@ -75,11 +75,11 @@ select key, sum2(val) filter (where valf < 5), sum2_strict(val) filter (where va
  key | sum2 | sum2_strict 
 -----+------+-------------
    1 |      |            
-   2 |      |          10
-   3 |      |            
-   5 |      |            
-   6 |      |            
-   7 |      |            
+   2 |   10 |          10
```

Comment:
hmz, why was this NULL before?

Reply:
Workers returned `NULL` instead of `0` when there were no rows passing filter, which then caused `NULL` to be combined instead of `0`